CODE_OF_CONDUCT.md

```diff
@@ -53,4 +53,4 @@ is deemed necessary and appropriate to the circumstances.
 
 Project maintainers who do not follow or enforce the Code of Conduct in good
 faith may face temporary or permanent repercussions as determined by other
-members of the project's leadership.
+members of the project's leadership.
```

Comment:
not sure why this line was changed, can you reset it to your dev branch? see [this answer on stackoverflow](https://stackoverflow.com/a/37972822). just change the branchname `master` to `dev`.